tools/dump_tool/ramdumpParser.py

```diff
@@ -645,9 +645,9 @@ def main():
 
 		# Find offset
 		global g_stext
-		g_stext = rParser.get_address_of_symbol("_stext")
+		g_stext = rParser.get_address_of_symbol("_stext_flash")
```

Comment:
Is it okay to check only flash?
There can be symbols in RAM also with some cases.